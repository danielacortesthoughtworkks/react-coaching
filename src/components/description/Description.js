import React from 'react';
import Typography from '@material-ui/core/Typography';
import 'typeface-roboto';

export const Description = (props) => {
  const { description } = props


  return (
    <Typography variant="h5" align="center">{description}</Typography>
  )
}

export default Description;
