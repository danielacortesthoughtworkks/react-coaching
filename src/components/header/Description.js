import React from 'react';

export const Description = (props) => {
  const { description } = props
  const divStyle = {
  width: '700px',
  position: 'absolute',
  left: '50%',
  right: '50%',
  transform: 'translate(-50%, 500%)'
  };

  return (

    <div style={divStyle}>
    <h5>{description}</h5>
    </div>
  )
}

export default Description;
