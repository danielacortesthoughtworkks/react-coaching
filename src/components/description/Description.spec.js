import React from 'react';
import { render } from '@testing-library/react';
import Description from './Description';


test('renders description', () => {
  const { getByText } = render(<Description description="Community driven roadmaps" />);
  const linkElement = getByText(/Community driven roadmaps/i);
  expect(linkElement).toBeInTheDocument();
});
