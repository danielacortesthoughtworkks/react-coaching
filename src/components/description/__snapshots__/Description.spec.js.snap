// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Description /> spec renders the component 1`] = `undefined`;
